api_tests/preprints/views/test_preprint_list.py

```diff
@@ -230,3 +235,12 @@ def test_file_not_osfstorage(self):
         assert_equal(res.status_code, 400)
         assert_equal(res.json['errors'][0]['detail'], 'This file is not a valid primary file for this preprint.')
 
+    def test_preprint_contributor_signal_sent_on_creation(self):
+        with capture_signals() as mock_signals:
+            public_project_payload = build_preprint_create_payload(self.public_project._id, self.subject._id,
+                                                                   self.file_one_public_project._id)
+            res = self.app.post_json_api(self.url, public_project_payload, auth=self.user.auth)
+
+            assert_equal(res.status_code, 201)
+            assert_equal(mock_signals.signals_sent(), set([project_signals.contributor_added]))
```

Comment:
Is it possible to check that this signal was sent for `self.other_user` and not sent for `self.user`?